src/server.ts

```diff
@@ -82,7 +85,7 @@ export class Server {
         ctx.respond = false
```

Comment:
I know it's not a change in this pull. But line 80 above `debug('yes!')` should be removed